### PR TITLE
Update 12-3.md

### PR DESCRIPTION
**Pull request by PeppeSaccardi**
Self-Attention Paragraph Typos: Issues #779

### CODE REVIEW

File: docs/es/week12/12-3.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ $$
 $$
 
 <!-- where $\boldsymbol{a} \in \mathbb{R}^n$ is a column vector with components $\alpha_i$. -->
```
Comment (Atcold) [inline docs/es/week12/12-3.md:49]:
```suggestion
<!-- where $\boldsymbol{a} \in \mathbb{R}^t$ is a column vector with components $\alpha_i$. -->
```

File: docs/fr/week12/12-3.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ $$
 \boldsymbol{h} = \boldsymbol{X} \boldsymbol{a}
 $$
 
-where $\boldsymbol{a} \in \mathbb{R}^n$ is a column vector with components $\alpha_i$.
+where $\boldsymbol{a} \in \mathbb{R}^t$ is a column vector with components $\alpha_i$.
```
Comment (Atcold) [inline docs/fr/week12/12-3.md:52]: This is the comment (which you correctly fixed), but you forgot to edit line 81 below.

File: docs/ja/week12/12-3.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ $$
 $$
 
 <!-- where $\boldsymbol{a} \in \mathbb{R}^n$ is a column vector with components $\alpha_i$. -->
```
Comment (Atcold) [inline docs/ja/week12/12-3.md:49]:
```suggestion
<!-- where $\boldsymbol{a} \in \mathbb{R}^t$ is a column vector with components $\alpha_i$. -->
```

**Review by Atcold — CHANGES_REQUESTED** [review]
Translations will have 2 typos, in the English comment and in the translated text.
Please, fix both, so that there are no discrepancies.

Thanks!

**Comment by PeppeSaccardi** [comment]
Thanks @Atcold! Now everything should be fixed!

**Review by Atcold — APPROVED** [review]
Great! Thanks so much!

**Pull request merged**